app/src/features/projects/view/__snapshots__/ProjectDetails.test.tsx.snap

```diff
@@ -229,9 +229,7 @@ exports[`ProjectDetails renders correctly 1`] = `
               >
                 <h6
                   class="MuiTypography-root MuiTypography-subtitle1"
-                >
-                  Placeholder
-                </h6>
```

Comment:
question: why has this placeholder been removed in the snaps?

Reply:
Fixed!